src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -13,8 +13,7 @@ public class ListCommand extends Command {
     public static final String COMMAND_WORD = "list";
 
     public static final String MESSAGE_SUCCESS = "Listed all participants";
-    public static final String MESSAGE_EMPTY = "No saved participants";
-
+    public static final String MESSAGE_EMPTY = "No saved participants to display";
```

Comment:
```suggestion
    public static final String MESSAGE_EMPTY = "No saved participants to display!";
```

Reply:
A lot of the messages do not end with a punctuation (! or .) so I'm inclined to keep it that way for now. It's a small preference decision anyways so if we want we can go back later to standardise the style later.